polkadot/node/network/bridge/src/rx/mod.rs

```diff
@@ -967,10 +968,12 @@ fn update_our_view<Context>(
 		finalized_number,
 	);
 
-	dispatch_validation_event_to_all_unbounded(
+	dispatch_validation_event_to_all(
```

Comment:
This has unintended consequences, right now the priority is:
1. Unbounded queue.
2. High Priority queue
3. Normal Priority queue.


What it means is that we process OurViewChange ahead of all the others `PeerConnected`/`PeerDisconnected`/`PeerViewChange`, that have been already queued.

But with this change we are going to put the OurViewChange in the same queue, so it mean all PeerViewChange already queued will be processed before OurViewChange.

I'm not saying something bad will happen, but I don't see the gain and I think most of our subsystem will benefit from processing OurViewChange as fast as it can.

Reply:
Yeah, this is a good point. We need an additional priority level to keep the priorities like we have them currently.

Reply:
I was also thinking about that. Because we already use the unbounded queue as the highest priority, we can mix the current order. But introducing one more level of priority can bring us a prioritization hell.

Reply:
Ideally we should never use unbounded for prioritization because of potential to introduce DOS vectors. but we should be fine as is without this PR - the number of `OurViewChange` messages is bounded by block production.